src/lib/grid-helper.ts

```diff
@@ -0,0 +1,28 @@
+import { get } from 'svelte/store';
+import { gridSettings } from './stores';
+import { getAvailablePosition } from './utils/grid';
+
+export class GridHelper {
+	static getFirstAvailablePosition(w: number, h: number) {
+		const gridParams = get(gridSettings);
+		if (!gridParams) throw new Error('Grid not initialized');
+		const { items, maxCols, maxRows } = gridParams;
+		return getAvailablePosition(
+			{
+				id: '',
+				x: 0,
+				y: 0,
+				w,
+				h,
+				movable: true,
+				resizable: true,
+				invalidate: () => {
+					/* .. */
+				}
+			},
+			Object.values(items),
+			maxCols,
+			maxRows
+		);
+	}
+}
```

Comment:
I dont like `GridHelper` name. For me it shoul be `GridConroller`, with mind of moving all methods from `utils/` to it in the future. 
ince gridParams should not be singleton, we can pass grid Params to constructor and then subsctribe store to controller instanse in Grid component.
Also I prefer PascalCase for class filenames: `GridConroller.ts`.

Reply:
Cool, i'll change it to GridController 🙌